Source/CCRakNetSlidingWindow.cpp

```diff
@@ -22,7 +22,7 @@ static const CCTimeType SYN=10000;
 
 #include "MTUSize.h"
 #include <stdio.h>
-#include <math.h>
+#include <cmath>

```

Comment:
As I commented also on https://github.com/OculusVR/RakNet/pull/64,
personally I would rather just change the `abs` to an `fabs` and avoid header changes like this.